app/services/teacher_training_public_api/sync_provider.rb

```diff
@@ -5,7 +5,7 @@ def initialize(provider_from_api:, recruitment_cycle_year:)
       @recruitment_cycle_year = recruitment_cycle_year
     end
 
-    def call(run_in_background: false)
+    def call(run_in_background: true)
```

Comment:
I think this should run in async as default. Let me know if you disagree 👍

Reply:
I don't disagree, but I think that was how it was configured on the old job. Is there any historical reason why it was set to `false`? @duncanjbrown